src/Form/UpdateMediaItemForVideoType.php

```diff
@@ -125,7 +123,7 @@ public function submitForm(array &$form, FormStateInterface $form_state) {
       // Up until here, all necessary checks have been made. No custom exception
       // handling needed other than for the db possibly exploding at this point.
       $this->messenger()->addError($this->t('There has been an unexpected problem updating the video. Check the logs for details.'));
-      $this->watchdogException($e);
+      $this->logger->watchdogException($e, sprintf('mpx video with guid %s could not be updated', $guid));
```

Comment:
I think this should be message and context variables without the sprintf, so the text can translated if needed?

Reply:
🤦‍♂  I misread the usage of the watchdogException method, and thought it wasn't passing the $variables for that. Changing.